examples/resources/gitlab_topic/import.sh

```diff
@@ -0,0 +1,4 @@
+# You can import a topic to terraform state using `terraform import <resource> <id>`.
+# The `id` must be an integer for the id of the topic you want to import,
+# for example:
+terraform terraform import gitlab_topic.functional-programming 1
```

Comment:
Let's use underscore in name. Also double terraform typo.
```suggestion
terraform import gitlab_topic.functional_programming 1
```